packages/client/css/generic/message-group.css

```diff
@@ -0,0 +1,37 @@
+.MessageGroup {
+  display: flex;
+
+  align-items: flex-start;
+  margin-bottom: 16px;
+
+  color: var(--gray-100);
+}
+
+.MessageGroup-authorAvatar {
+  width: 48px;
+  height: 48px;
+
+  margin-right: 16px;
+  border-radius: 4px;
+}
+
+.MessageGroup-info {
+  margin-bottom: 4px;
+
+  & time {
+    display: inline-block;
+    color: var(--gray-300);
+  }
+}
+
+.MessageGroup-authorUsername {
+  display: inline-block;
+  margin-right: 4px;
+
+  font-weight: bold;
+  text-align: right;
+}
+
+.MessageGroup-contents {
+  width: calc(100% - 48px - 16px);
```

Comment:
What is the reasoning for these numbers? I love separating out `48px` and `16px` (as opposed to `100% - 32px`) if they have distinct meanings, but it's not clear from the code what those meanings are.

A simple comment should clear things up.

Reply:
Gosh, yeah. @heyitsmeuralex I absolutely can't stand using `calc`. Could you rework this with flexbox, or even just float?

Reply:
aaaa No idea what this line is for :package:
I'll look into it.